ansible/roles/liquid-core/templates/libexec/create-oauth-application

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -e
+
+if [ "$#" -ne 2 ]; then
+    echo "Usage: create-oauth-application APP_NAME REDIRECT_URI"
+    echo "The script will fill out keys under /var/lib/liquid/oauth_keys/\$APP_NAME."
+    exit 1
+fi
+
+APP_NAME=$1
+REDIRECT_URI=$2
+
+/opt/liquid-core/venv/bin/python /opt/liquid-core/liquid-core/manage.py shell <<EOF
```

Comment:
This looks like a great candidate for a liquid-core management command. Shall we make a separate issue?

Reply:
Yeah, separate issue. I'll make it.

Reply:
See https://github.com/liquidinvestigations/liquidinvestigations/issues/91